mne/viz/epochs.py

```diff
@@ -953,6 +956,27 @@ def _plot_traces(params):
         ax.set_yticklabels(labels, fontsize=12, color='black')
     else:
         ax.set_yticklabels(tick_list, fontsize=12)
+
+    if len(params['events']) > 0:  # vertical lines for events.
+        for ev_line in params['ev_lines']:
+            ax.lines.remove(ev_line)  # clear the view first
+        params['ev_lines'] = list()
+        t_zero = np.where(epochs.times == 0.)[0]  # idx of 0s
```

Comment:
that would not distinguish epochs with and without zero-point (for example: first epoch sample one second after event). In cases when zero-point is present - is it possible that it is not exactly zero? If so - maybe this could look for `np.abs(epoch.times) <= half_sample_time`?